### DISCUSSION TITLE
Windows版讨论区

### DISCUSSION THREAD

**Discussion by caol64** [caol64/wenyan#12] [General]
Windows版功能、进度、答疑讨论。

下载地址：https://yuzhi.tech/docs/wenyan/windows

Windows版已开源：https://github.com/caol64/wenyan-pc

**Comment by caol64** [comment]
## Windows v2.0发布

-   新增自定义主题功能
-   新增导出长图功能（beta版）
-   优化公式显示

https://yuzhi.tech/docs/wenyan/windows

**Comment by caol64** [comment]
## v2.1

-   自定义主题功能优化

**Comment by caol64** [comment]
## v2.2（已开源）

- 开放内置主题，可以在内置主题的基础上生成自定义主题
- 支持导入外部`CSS`转换为自定义主题
- 支持文件拖拽打开功能
- 公式显示优化

**Reply by zw-95** [reply]
windows版2.2，导入功能无效
![PixPin_2024-12-06_11-02-43](https://github.com/user-attachments/assets/00536283-2a35-4d0d-b2bd-6cb344beac1e)

**Reply by caol64** [reply]
我猜测是字符编码问题，你把编码改成utf8试下。如果确定是这个问题，请提一个issue。

**Reply by zw-95** [reply]
> 我猜测是字符编码问题，你把编码改成utf8试下。如果确定是这个问题，请提一个issue。

确实，改为tuf8就行了，我保存的是不常见的utf-16

**Comment by caol64** [comment]
## v2.2.1

-  修复公众号格式丢失问题

**Comment by caol64** [comment]
本帖不再通告版本更新，版本更新记录请查阅：https://yuzhi.tech/docs/wenyan/windows
